src/tools/qt-gui/src/guisettings.cpp

```diff
@@ -141,6 +165,22 @@ void GUISettings::lookupColor (const std::string & keyName, QColor & toSet) cons
 	}
 }
 
+void GUISettings::lookupBool (const std::string & keyName, bool & value) const
+{
+	Key key = m_config.lookup (keyName);
+
+	if (!key) return; // nothing to do
+
+	try
+	{
+		value = (key.get<std::string> ().compare ("true") == 0 ? true : false);
```

Comment:
Again, what about `.get<bool>`?

Unfortunately #308 is not implemented yet (a plugin that should bring different ways of saying true/false such as on/off to a canonical form)